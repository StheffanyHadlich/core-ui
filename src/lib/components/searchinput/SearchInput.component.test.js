import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/searchinput/__snapshots__"
});
