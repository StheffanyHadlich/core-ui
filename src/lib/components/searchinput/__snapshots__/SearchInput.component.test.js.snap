// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots SearchInput Default 1`] = `
<div>
  <h3>
    Default
  </h3>
  <div
    style={
      Object {
        "width": "250px",
      }
    }
  >
    <div
      className="sc-searchinput sc-gzVnrw kjCCUz"
    >
      <div
        className="sc-input sc-bxivhb ktEkRb"
      >
        <div
          className="sc-input-wrapper sc-bZQynM AMncx"
        >
          <input
            className="sc-input-type"
            name="search"
            onBlur={[Function]}
            onChange={[Function]}
            onKeyDown={[Function]}
            placeholder="Search server..."
            type="text"
            value=""
          />
        </div>
      </div>
      <button
        className="sc-htoDjs sc-dnqmqq bGFxWO"
        disabled={false}
        onClick={[Function]}
      >
        <i
          className="fas fa-search"
        />
      </button>
      <button
        className="sc-htoDjs sc-iwsKbI dBQYwg"
        onClick={[Function]}
      >
        <i
          className="fas fa-times-circle"
        />
      </button>
    </div>
  </div>
  <h3>
    Search Input filled
  </h3>
  <div
    style={
      Object {
        "width": "250px",
      }
    }
  >
    <div
      className="sc-searchinput sc-gzVnrw kjCCUz"
      data-cy="carlito_searchinput"
    >
      <div
        className="sc-input sc-bxivhb ktEkRb"
      >
        <div
          className="sc-input-wrapper sc-bZQynM AMncx"
        >
          <input
            className="sc-input-type"
            name="search"
            onBlur={[Function]}
            onChange={[Function]}
            onKeyDown={[Function]}
            type="text"
            value="carlito"
          />
        </div>
      </div>
      <button
        className="sc-htoDjs sc-dnqmqq bGFxWO"
        disabled={false}
        onClick={[Function]}
      >
        <i
          className="fas fa-search"
        />
      </button>
      <button
        className="sc-htoDjs sc-iwsKbI dBQYwg"
        onClick={[Function]}
      >
        <i
          className="fas fa-times-circle"
        />
      </button>
    </div>
  </div>
  <h3>
    Search Input disable default toggle
  </h3>
  <div
    style={
      Object {
        "width": "250px",
      }
    }
  >
    <div
      className="sc-searchinput sc-gzVnrw hqmodQ"
    >
      <div
        className="sc-input sc-bxivhb ktEkRb"
      >
        <div
          className="sc-input-wrapper sc-bZQynM AMncx"
        >
          <input
            className="sc-input-type"
            name="search"
            onBlur={[Function]}
            onChange={[Function]}
            onKeyDown={[Function]}
            placeholder="Search and Filter..."
            type="text"
            value=""
          />
        </div>
      </div>
      <button
        className="sc-htoDjs sc-dnqmqq fzqjWe"
        disabled={true}
        onClick={[Function]}
      >
        <i
          className="fas fa-search"
        />
      </button>
      <button
        className="sc-htoDjs sc-iwsKbI dBQYwg"
        onClick={[Function]}
      >
        <i
          className="fas fa-times-circle"
        />
      </button>
    </div>
  </div>
</div>
`;
